README.md

```
@@ -211,25 +217,44 @@ Use the search tool to find information about "latest developments in AI"
 
 ```
 Search for "climate change solutions" using:
-- Model: perplexity/sonar
+- Model: sonar-pro
 - Max tokens: 2000
 - Temperature: 0.3
 ```
 
+### Using Different Models
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)